src/theme/base/typography.ts

```diff
@@ -0,0 +1,42 @@
+const typography = {
+  fontWeights: {
+    hairline: 100,
+    thin: 200,
+    light: 300,
+    normal: 400,
+    medium: 500,
+    semibold: 600,
+    bold: 700,
+    extrabold: 800,
+    black: 900,
+    extraBlack: 950,
```

Comment:
Does fontWeight work for android?

Reply:
it is work both android and ios

Reply:
fontWeight probably doesn't work on Android. we just should use fontFamily.